docs/plugins/autoslab.rst

```diff
@@ -0,0 +1,20 @@
+autoslab
+========
+
+.. dfhack-tool::
+    :summary: Automatically engrave slabs for ghostly citizens!
```

Comment:
summaries must actually end in a period to conform with the in-game help text parser